streams/src/test/java/org/apache/kafka/streams/integration/AbstractResetIntegrationTest.java

```diff
@@ -537,15 +576,15 @@ private Topology setupTopologyWithoutIntermediateUserTopic() {
         return builder.build();
     }
 
-    private void cleanGlobal(final boolean withIntermediateTopics,
-                             final String resetScenario,
-                             final String resetScenarioArg) throws Exception {
+    private int tryCleanGlobal(final boolean withIntermediateTopics,
```

Comment:
No logic change here, just extract the main body of cleanGlobal out and put it in tryCleanGlobal.